modules/seo.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Module Name: seo
+ * Module Description: Adds tools to enhance your site's content for better results on search engines and social media.
+ * Sort Order: 35
+ * First Introduced: 4.3
+ * Requires Connection: No
+ * Auto Activate: Yes
+ * Module Tags: Social, Appearance
+ */
+
+include dirname( __FILE__ ) . '/seo/helpers-seo.php';
+include dirname( __FILE__ ) . '/seo/title-seo.php';
+include dirname( __FILE__ ) . '/seo/posts-seo.php';
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class SEO {
+	function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	function init() {
+		add_action( 'wp_head', array( $this, 'meta_tags' ) );
```

Comment:
I wonder if it would be worth leveraging our existing Open Graph structure, and hook all new meta tags into `jetpack_open_graph_tags` instead. That would give people access to all the filters available there, to customize meta tags, or remove them when necessary. It would also allow us to leverage the existing work we've made with other SEO plugins, to make sure we're fully compatible (i.e. when Yoast SEO is active, for example, the tags won't be added).